pages/common/docker-run.md

```diff
@@ -0,0 +1,28 @@
+# docker run
+
+> Run a command in a new container.
```

Comment:
```suggestion
> Run a command in a new Docker container.
```

Reply:
Is it really relevant to specify “Docker container” since the command `docker run` already implies we are working with Docker objects ?